redisvl/extensions/llmcache/semantic.py

```diff
@@ -183,6 +191,20 @@ def delete(self) -> None:
         index."""
         self._index.delete(drop=True)
 
+    def drop(self, document_ids: Union[str, List[str]]) -> None:
+        """Remove a specific entry or entries from the cache by it's ID.
+
+        Args:
+            document_ids (Union[str, List[str]]): The document ID or IDs to remove from the cache.
+        """
+        if isinstance(document_ids, List):
```

Comment:
In a similar theme to the work around `clear()`, to me this feels like an addition to that method by supporting an optional list of ids? by default clear removes everything. Maybe if this list is provided, instead of clearing everything it will just focus on using this pipeline code to clear the subset?

Reply:
I like the idea of combining methods. What about something like 
```
clear(keys: Optional[List[str], str]):
"""Remove all or specific entries from the cache by it's ID.
    Args:
        document_ids (Union[str, List[str]]): The document ID or IDs to remove from the cache. If none are provided then all documents are removed.
"""
```

Reply:
Will move this pipeline code into index.